docs/weather-skills.md

```diff
@@ -75,61 +89,4 @@ aea delete my_weather_station
 aea delete my_weather_client
 ```
 
-## Using the ledger
```

Comment:
@katharine-murphy why has this been removed?

Reply:
I think because I pulled from `master` rather than `develop`.